File: VB/Customer.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace T263441
    Public Class Customer
        Private _firstName As String
        Private _lastName As String
        Private _country As String
        Private _hasChildren As Boolean
        Private _cash As Integer

        Public Sub New()
        End Sub

        Public Property FirstName() As String
            Get
                Return _firstName
            End Get
            Set(ByVal value As String)
                If value <> _firstName Then
                    _firstName = value
                End If
            End Set
        End Property
        Public Property LastName() As String
            Get
                Return _lastName
            End Get
            Set(ByVal value As String)
                If value <> _lastName Then
                    _lastName = value
                End If
            End Set
        End Property

        Public Property Country() As String
            Get
                Return _country
            End Get
            Set(ByVal value As String)
                If value <> _country Then
                    _country = value
                End If
            End Set
        End Property

        Public Property HasChildren() As Boolean
            Get
                Return _hasChildren
            End Get
            Set(ByVal value As Boolean)
                If value <> _hasChildren Then
                    _hasChildren = value
                End If
            End Set
        End Property

        Public Property Cash() As Integer
            Get
                Return _cash
            End Get
            Set(ByVal value As Integer)
                If value <> _cash Then
                    _cash = value
                End If
            End Set
        End Property
    End Class
End Namespace

File: CS/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace T263441 {
    public class Customer {
        string _firstName;
        string _lastName;
        string _country;
        bool _hasChildren;
        int _cash;

        public Customer() {
        }

        public string FirstName {
            get { return _firstName; }
            set {
                if(value != _firstName) {
                    _firstName = value;
                }
            }
        }
        public string LastName {
            get { return _lastName; }
            set {
                if(value != _lastName) {
                    _lastName = value;
                }
            }
        }

        public string Country {
            get { return _country; }
            set {
                if(value != _country) {
                    _country = value;
                }
            }
        }

        public bool HasChildren {
            get { return _hasChildren; }
            set {
                if(value != _hasChildren) {
                    _hasChildren = value;
                }
            }
        }

        public int Cash {
            get { return _cash; }
            set {
                if(value != _cash) {
                    _cash = value;
                }
            }
        }
    }
}
